src/rapids_dependency_file_generator/rapids_dependency_file_generator.py

```diff
@@ -165,9 +165,29 @@ def make_dependency_file(
                 table[section] = tomlkit.table()
                 table = table[section]
 
-        key = extras.get(
-            "key", "requires" if extras["table"] == "build-system" else "dependencies"
-        )
+        if extras["table"] == "build-system":
+            key = "requires"
+            if key in extras:
```

Comment:
Should this have been:

```suggestion
            if "key" in extras:
````

?

Reply:
Yes, this is definitely a typo. Feel free to fix in your next PR. The same issue also exists on line 176 here.